src/components/input/PasswordInput.tsx

```diff
@@ -0,0 +1,42 @@
+'use client';
+
+import * as React from 'react';
```

Comment:
```suggestion
import { useState, forwardRef } from 'react';
```
공식 문서에서도 * 대신 명시적으로 import 하네요! :)
https://react.dev/reference/react/forwardRef#forwardref